rllib/algorithms/ppo/torch/ppo_torch_rl_module.py

```diff
@@ -90,6 +90,10 @@ def _compute_values(self, batch, device=None):
 
         # Separate vf-encoder.
         if hasattr(self.encoder, "critic_encoder"):
+            if self.is_stateful():
```

Comment:
Awesome! Ran into this issue yesterday as well (and continued testing then with a shared value function :) ).